imports/plugins/included/shipping-shippo/server/methods/shippo.js

```diff
@@ -30,6 +30,29 @@ function createShippoAddress(reactionAddress, email, purpose) {
   return shippoAddress;
 }
 
+function createValidatedAddress(apiResult) {
+  const formErrors = [];
+  if (apiResult.messages) {
+    apiResult.messages.forEach((message) => {
+      formErrors.push({
+        summary: message.code,
+        details: message.text
+      });
+    });
+  }
+  return {
+    validatedAddress: {
+      address1: apiResult.street1,
+      address2: apiResult.street2,
```

Comment:
Does this always return a result for `street2` or do we need to protect against this being `undefined`?

Reply:
@zenweasel No this won't always return a result but even it doesn't the `address2` will be undefined, which is not a problem.